catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -332,6 +332,12 @@ public UpdateResponse update(UpdateRequest updateRequest) throws IngestException
         String attributeQuery = getQuery(attributeName, identifiers);
 
         SolrQuery query = new SolrQuery(attributeQuery);
+        // Set number of rows to the result size + 1.  The default row size in Solr is 10, so this
+        // needs to be set in situations where the number of metacards to update is > 10.  Since there
+        // could be more results in the query resposne than the number of metacards in the update request,
+        // 1 is added to the row size, so we can still determine whether we found more metacards than
+        // updated metacards provided
+        query.setRows(updates.size() + 1);
```

Comment:
Not sure why a larger row size would be better.  If more than update.size() is returned, we know something is wrong with the request.